packages/eds-core-react/src/components/SideBar/SideBarAccordionItem/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { forwardRef, AnchorHTMLAttributes, ElementType } from 'react'
+import styled, { css } from 'styled-components'
+import { OverridableComponent } from '@equinor/eds-utils'
+
+import { Typography } from '../../Typography'
+import { sidebar as tokens } from '../SideBar.tokens'
+import { useSideBar } from '../SideBar.context'
+
+const {
+  minWidth,
+  entities: {
+    sidebarItem: {
+      minHeight,
+      typography: { color: typographyColor },
+      states: {
+        active: {
+          background: menuActiveBackground,
+          typography: { color: typographyActiveColor },
+        },
+        hover: { background: menuHoverBackground },
+      },
+    },
+  },
+} = tokens
+
+const Container = styled.div<{ $active?: boolean }>(({ $active }) => {
+  return css`
+    display: grid;
```

Comment:
Need to add some css to reset some potential default styles since it is polymorphic. 
`text-decoration: none;` to prevent this:
![bilde](https://user-images.githubusercontent.com/4499531/227945664-a6f60874-e923-4372-993e-0769d30fc710.png)
And then `border: 0; width: 100%; padding: 0; ` background-color transparent instead of none to prevent this (as="button"):
![bilde](https://user-images.githubusercontent.com/4499531/227946226-0de96dd6-000b-4f17-b988-1dd6f6d21721.png)

We are also missing the focus outline for eds (copy it from sidebarLink)